vita3k/nids/include/nids/nids.inc

```diff
@@ -2044,6 +2044,25 @@ NID(sceImeSetCaret, 0xD8342D2A)
 NID(sceImeSetPreeditGeometry, 0x7B1EFAA5)
 NID(sceImeSetText, 0xF3BD9A76)
 NID(sceImeUpdate, 0x71D6898A)
+// Library "SceIpmi"
+NID(_ZN4IPMI6Client10disconnectEv, 0xEC73331C)
+NID(_ZN4IPMI6Client11getUserDataEv, 0x004F48ED)
+NID(_ZN4IPMI6Client12tryGetResultEjPiPvPmm, 0xF8C2B8BA)
+NID(_ZN4IPMI6Client12tryGetResultEjjPiPNS_10BufferInfoEj, 0x4EBB01A2)
+NID(_ZN4IPMI6Client13pollEventFlagEjjjPj, 0x8FF23C3C)
+NID(_ZN4IPMI6Client13waitEventFlagEjjjPjS1_, 0x45C32034)
+NID(_ZN4IPMI6Client16invokeSyncMethodEjPKNS_8DataInfoEjPiPNS_10BufferInfoEj, 0x73C72FBB)
+NID(_ZN4IPMI6Client16invokeSyncMethodEjPKvjPiPvPjj, 0x28BD5F19)
+NID(_ZN4IPMI6Client17invokeAsyncMethodEjPKNS_8DataInfoEjPjPKNS0_12EventNotifeeE, 0x387AFA3F)
+NID(_ZN4IPMI6Client17invokeAsyncMethodEjPKvjPiPKNS0_12EventNotifeeE, 0xAFD10F3B)
+NID(_ZN4IPMI6Client19terminateConnectionEv, 0xD484D36D)
+NID(_ZN4IPMI6Client6Config24estimateClientMemorySizeEv, 0x4E255C31)
+NID(_ZN4IPMI6Client6createEPPS0_PKNS0_6ConfigEPvS6_, 0xB282B430)
+NID(_ZN4IPMI6Client6getMsgEjPvPjjS2_, 0xA3E650B0)
+NID(_ZN4IPMI6Client7connectEPKvjPi, 0xA22C3E01)
+NID(_ZN4IPMI6Client7destroyEv, 0x101C93F8)
+NID(_ZN4IPMI6Client9tryGetMsgEjPvPmm, 0x60EFADE7)
+NID(_ZN4IPMI6ClientD1Ev, 0xA5AA193C)
```

Comment:
It should be worth mentionning some of them are "fake" nids

Reply:
Nid is not sur ? 🤔

Reply:
I have edited and added comment for nids